control-plane/connect-inject/controllers/pod/pod_controller.go

```diff
@@ -762,3 +764,11 @@ func getDestinationsID(name, namespace, partition string) *pbresource.ID {
 		},
 	}
 }
+
+func getLogFieldsForResource(id *pbresource.ID) []any {
```

Comment:
Might be helpful to move this out into a commonly-used package at some point but didn't do this now